bokehjs/src/coffee/models/annotations/index.ts

```diff
@@ -1,20 +1,21 @@
-export {Annotation}     from "./annotation"
-export {Arrow}          from "./arrow"
-export {ArrowHead}      from "./arrow_head"
-export {OpenHead}       from "./arrow_head"
-export {NormalHead}     from "./arrow_head"
-export {TeeHead}        from "./arrow_head"
-export {VeeHead}        from "./arrow_head"
-export {Band}           from "./band"
-export {BoxAnnotation}  from "./box_annotation"
-export {ColorBar}       from "./color_bar"
-export {Label}          from "./label"
-export {LabelSet}       from "./label_set"
-export {Legend}         from "./legend"
-export {LegendItem}     from "./legend_item"
-export {PolyAnnotation} from "./poly_annotation"
-export {Span}           from "./span"
-export {TextAnnotation} from "./text_annotation"
-export {Title}          from "./title"
-export {Tooltip}        from "./tooltip"
-export {Whisker}        from "./whisker"
+export {Annotation}        from "./annotation"
+export {Arrow}             from "./arrow"
+export {ArrowHead}         from "./arrow_head"
+export {OpenHead}          from "./arrow_head"
+export {NormalHead}        from "./arrow_head"
+export {TeeHead}           from "./arrow_head"
+export {VeeHead}           from "./arrow_head"
+export {Band}              from "./band"
+export {BoxAnnotation}     from "./box_annotation"
+export {ColorBar}          from "./color_bar"
+export {Label}             from "./label"
+export {LabelSet}          from "./label_set"
+export {Legend}            from "./legend"
+export {LegendItem}        from "./legend_item"
+export {PolyAnnotation}    from "./poly_annotation"
+export {Span}              from "./span"
+export {TextAnnotation}    from "./text_annotation"
+export {Title}             from "./title"
+export {ToolbarAnnotation} from "./toolbar_annotation"
```

Comment:
I'm OK with `Toolbar` inheriting from `Annotation` if it must be (tho I'd love to fix it sometime), but we should just call it `Toolbar` not `ToolbarAnnotation` even so

Reply:
I need to figure out how to do side panels anyway, and thus annotations, so this will be done. However, I could rename it now to e.g. `ToolbarPanel` (after all, there is already `ToolbarBox` for detached toolbars). Renaming to just `Toolbar` would mean renaming actual `Toolbar` to something else. Currently we can't have two models with the same name.